composer.json

```diff
@@ -37,17 +37,17 @@
     ],
 
     "require": {
-        "php": ">=5.5",
-        "league/climate": "^2.0|^3.0",
+        "php": "^5.6 || ^7.0",
```

Comment:
Huh, funny. I've only ever used a single pipe and it worked. You're absolutely right about the [docs](https://getcomposer.org/doc/articles/versions.md#range) though.